### ISSUE TITLE
IRC bridge mishandles CTCP Actions (/me) in IRC channel

### ISSUE THREAD

**Issue by dopeghoti** [kharidiron/StarryPy3k#27]
As pictured below, IRC actions (e. g. `/me does a thing`) are handled poorly by the IRC bridge.  Presumably other CTCP events to the channel would be similarly mishandled.

![starbound irc relay bug](https://cloud.githubusercontent.com/assets/295727/17378399/a5f9b3e4-5972-11e6-9040-0f753d14d47e.png)

**Comment by kharidiron** [comment]
Hm... yeah, the `irc_bot` plugin is not setup to handle the CTCP responses. I need to educate myself more in IRC implementation to check how to fix this, but I think it should be pretty straight-forward.

**Comment by dopeghoti** [comment]
The simple way to handle this is some form of:

```
for line in irc.connection.readline():
   parts=line.split()
   if len( parts ) > 2:
      if parts[2][0] == "\x01":        # This is a CTCP event
         if parts[2] == "\x01ACTION":  # This is a CTCP Action (e. g. `/me does a thing`)
             output = " * " + parts[1].replace('<','').replace('>','') + " " + join ( parts[3:] )  # strip < and > from around the actor's nick
         else: 
            pass      # until other CTCP events are handled
      else: 
         pass         # normal message, handle as previously

```

**Comment by kharidiron** [comment]
Is this something easy enough to make a PR of? If so, would you be willing to do so? :3

**Comment by dopeghoti** [comment]
I'll see what I can do; I'll do some hacking and testing tonight.

**Comment by kharidiron** [comment]
Resolved in #32. Will reopen if I'm wrong.
